Header header

string object_name
geometry_msgs/Pose object_pose
float64 log_likelihood

uint32 requestID